questions/questions.js

```diff
@@ -23,18 +23,95 @@ function getStartingQuestions() {
   });
 }
 
-console.log(getStartingQuestions(startingQuestions));
+const subjects = ["Math", "English", "Javascript"];
+const skillLevels = [1, 2, 3, 4, 5];
 
-// const skillLevel = ["Elementary", "Intermediate", "Advanced", "Proficient"];
+const chosenSkillLevelAnswers = {
+  math: {
+    chosenAnswer: null,
+  },
+  english: {
+    chosenAnswer: null,
+  },
+  javascript: {
+    chosenAnswer: null,
+  },
+};
 
-// const radioButton = document.createElement("input");
-// radioButton.setAttribute("type", "radio");
-// radioButton.value = optionValue;
+window.onload = () => {
+  renderStartingQuestions(startingQuestions);
+  renderSkillsLevelQuiz(skillLevels, subjects);
+};
 
-// const label = document.createElement("label");
-// label.innerHTML = optionValue;
+const formElem = document.querySelector("form");
+const skillBox = document.createElement("ul");
+skillBox.classList.add("choices");
 
-// radio buttons
-//Math
-//English
-//JS
+formElem.addEventListener("submit", (event) => {
+  event.preventDefault();
+});
+
+// handleMathButtonClick() also for JS and English
+// higher-order function name as a parameter
+// makeButtonClickHandler()
+
+function makeButtonClickHandler() {
```

Comment:
subject needs to be passed to the makeButtonClickHandler so that it will close over the inner function